{"name": "Email -- @1.1 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=87.0.4280.88)\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/behave/runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"features/environment.py\", line 78, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(),\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 1064, in get_screenshot_as_png\n    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 1074, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)['value']\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given clear time", "status": "passed", "start": 1608635524763, "stop": 1608635524763}, {"name": "When open f.gum24.online page", "status": "passed", "start": 1608635524764, "stop": 1608635530212}, {"name": "When clear time", "status": "passed", "start": 1608635530213, "stop": 1608635530213}, {"name": "When go to confirmation page", "status": "passed", "start": 1608635530213, "stop": 1608635534208}, {"name": "When calculate time for 'confirmation'", "status": "passed", "start": 1608635534208, "stop": 1608635534531}, {"name": "When go to cloudpayments page", "status": "passed", "start": 1608635534531, "stop": 1608635535011}, {"name": "When enter fake card details", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=87.0.4280.88)\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/behave/runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"features/environment.py\", line 78, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(),\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 1064, in get_screenshot_as_png\n    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 1074, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)['value']\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1608635535011, "stop": 1608635535015}, {"name": "Then I see cancel button element", "status": "skipped", "start": 1608635535016, "stop": 1608635535016}, {"name": "When calculate time for 'cloudpayments'", "status": "skipped", "start": 1608635535016, "stop": 1608635535016}], "parameters": [{"name": "page_name", "value": "f.gum24.online"}, {"name": "card", "value": "fake card"}, {"name": "type", "value": "cancel button"}], "start": 1608635524761, "stop": 1608635535016, "uuid": "d0f1b28e-b9a6-4861-a5ed-e30bab1c71e0", "historyId": "055759470ca507e62b8429e2c3fe0615", "fullName": "features/tests/4/cloudpayments7.feature:Email", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "validation"}, {"name": "feature", "value": "fitness-online.ru.com"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}